### DISCUSSION TITLE
Foundry Fundamentals - Section1 SimpleStorage / Video Error

### DISCUSSION THREAD

**Discussion by Snaporaz6** [Cyfrin/foundry-full-course-cu#1944] [General]
Hi Cyfrin! I get confused in this part of the course and then I found out it was for the fact lesson16: .Env Bye and Lesson 20: Never Use A Env File have the same video, ( I guess should be only for 20 ). Also lesson 16 have no text in "written lesson".                                                      

Also in "Lesson 17: Deploy a smart contract locally using Anvil" the video dosn't match the text, in the video is deployed "DeployFundMe.s.sol" and in the text "DepoloySimpleStorage.s.sol".   

![Screenshot 2024-07-04 165840](https://github.com/Cyfrin/foundry-full-course-cu/assets/83072437/b746fa69-9ea3-4759-aa67-2c7fb0d7a46a)
![Screenshot 2024-07-04 170241](https://github.com/Cyfrin/foundry-full-course-cu/assets/83072437/4f73c946-5548-4bde-aff5-4faacec664be)

**Comment by EngrPips** [comment]
Hello @Snaporaz6, Can you please use the issue tab of this Github repo to report these issues?

**Reply by Snaporaz6** [reply]
Sorry! 🙃

**Reply by EngrPips** [reply]
It's Fine, Friend.

**Comment by cromewar** [comment]
Hello @Snaporaz6, thanks for pointing this out; the corrections were made for lesson 17, and lesson 16 is still in progress.
